### DISCUSSION TITLE
List of features, fixes or changes as compared to Monogame

### DISCUSSION THREAD

**Discussion by Epicguru** [kniEngine/kni#1689] [Ideas & Suggestions]
It would be nice if there were a general overview of the differences or improvements that KNI offers compared to Monogame. Based on the changelog and pull requests I can see that there is a lot of activity going on, but as a potential new user it seems that I would have to dig through the codebase to understand how KNI differs from Monogame.

Thanks.

**Comment by vchelaru** [comment]
I don't have an answer to all of the features, but here are the big reasons why I use Kni:

1. WebGL support - this is the biggest one for me
2. Better performance, as shown on release blogs
3. A more responsive maintainer - despite its funding and larger community MonoGame is still struggling to find a "champion" who will push the tech forward. By contrast nkast is super active and responds to bugs/issues far more quickly than MonoGame.
